src/WattleScript.Tests/EndToEnd/CLikeTestRunner.cs

```diff
@@ -45,6 +45,7 @@ public async Task RunCore(string path, bool reportErrors = false)
         StringBuilder stdOut = new StringBuilder();
 
         Script script = new Script(CoreModules.Preset_HardSandboxWattle | CoreModules.PrototypeTable);
+        script.Options.InstructionLimit = ulong.MaxValue;
```

Comment:
Why?

Reply:
Tested if it crashed correctly with `while(true) {print("something")}`. Can it stay this way?